cmd/fleetctl/package.go

```diff
@@ -241,6 +260,23 @@ To add other devices to Fleet, distribute this installer using Chef, Ansible, Ja
 	}
 }
 
+func shouldRetry(pkgType string, opt packaging.Options, err error) bool {
```

Comment:
no change requested, I know this was specified like this, but just want to share: I _believe_ (no evidence) the issue is not strictly related to macOS, but rather to using Wine in `arm64` in an `amd64` image (`fleetdm/wix`).

Reply:
Ok, yeah if we find there are other conditions that should trigger a retry we can always add support for them here.